File: backend/src/main/java/com/instantmechanic/mechanic_api/model/Mechanic.java
package com.instantmechanic.mechanic_api.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Mechanic {

    private Long id;
    private String name;
    private Double rating;
    private String distance;
    private String location;
    private List<String> services;
    private Boolean isOpen;
    private String address;
    private String workingHours;
    private String phone;
}

File: app/src/main/java/com/example/instantmechanic/data/model/Mechanic.kt
package com.example.instantmechanic.data.model

data class Mechanic(
    val id: Long,
    val name: String,
    val rating: Double,
    val distance: String,
    val location: String,
    val services: List<String>,
    val isOpen: Boolean,
    val address: String,
    val workingHours: String,
    val phone: String
)